schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/layout/config-editor.xml

```diff
@@ -437,8 +438,8 @@
       </sidePanel>
 
       <tab id="default" default="true" mode="flat">
-        <section xpath="/mdb:MD_Metadata/mdb:identificationInfo"/>
-        <section xpath="/mdb:MD_Metadata/mdb:contentInfo"/>
+        <section xpath="/mdb:MD_Metadata/mdb:identificationInfo" collapsible="false"/>
```

Comment:
Shouldn't we apply the same default configuration for ISO19139 schema ?

Reply:
+1 for me, especially the 'simplify default editor' one, make one with only the fields needed to validate

Reply:
> Shouldn't we apply the same default configuration for ISO19139 schema ?

Done.